nano/node/bootstrap_ascending/service.cpp

```diff
@@ -132,7 +132,7 @@ void nano::bootstrap_ascending::service::inspect (store::transaction const & tx,
 	{
 		case nano::block_status::progress:
 		{
-			const auto account = ledger.account (tx, hash);
+			const auto account = ledger.account (block);
```

Comment:
Not reusing existing db transaction is a big red flag, given this is called very frequently, what's the performance impact here?